Tasks/AzureAppServiceManage/azureappservicemanage.ts

```diff
@@ -34,7 +34,7 @@ async function run() {
         var errorMessage: string = "";
         var updateDeploymentStatus: boolean = true;
         var azureEndpoint: AzureEndpoint = await new AzureRMEndpoint(connectedServiceName).getEndpoint();
-
+        console.log("##vso[telemetry.publish area=TaskEndpointId;feature=AzureAppServiceManage]" + '{"endpointId":"' + connectedServiceName + '"}');
```

Comment:
nit: Can we use JSON.stringify to create the JSON string for Xplat tasks?

Reply:
Also, please ensure that the tasks (modified here) has a higher patch version than Releases/M133

Reply:
Yes it has.